docs/nextflow-create-docker.md

```diff
@@ -98,7 +99,17 @@ In our example here, we will have you build your image using a `requirements.txt
 
 Our example will use the files in the [torch_cuda_test directory](https://github.com/uc-cdis/bio-nextflow/tree/master/nextflow_notebooks/containerized_gpu_workflows/torch_cuda_test) of the bio-nextflow repository. You can review the `readme` file in this directory for more information. It is a simple example that will build up from our base image by adding PyTorch. The Nextflow script will ultimately use a python script that checks the version of CUDA in the GPU instance and checks whether it is compatible with the version of PyTorch and CUDA available in the container.
 
-First, in the terminal, navigate to the directory where the downloaded Dockerfile and requirements.txt are located.
+First, download the contents of the [torch_cuda_test directory](https://github.com/uc-cdis/bio-nextflow/tree/master/nextflow_notebooks/containerized_gpu_workflows/torch_cuda_test). If you have `git`, the easiest way is to clone the repository:
+```
+git clone git@github.com:uc-cdis/bio-nextflow.git
+or
+git clone https://github.com/uc-cdis/bio-nextflow.git
+```
+
+Then, in the terminal, navigate to the directory where the downloaded Dockerfile and requirements.txt are located.
+```
+cd bio-nextflow/nextflow_notebooks/containerized_gpu_workflows/torch_cuda_test
+```
 
 > Note that the first line of the Dockerfile references the fetch tag for one of our GPU base images. This is always how you will reference a base image -- with `FROM` and the Dockertag.
```

Comment:
@paulineribeyre -- do you know if this is accurate? I see some Dockerfiles use the FROM, but I think Aarti had told me that the "docker pull" command that you copy in the fetch tag window is what is going to be the first line in the dockerfile. Do you use BOTH the "docker pull" and the FROM? or are they interchangeable?

Reply:
i'm not sure what she meant but the dockerfile contains "FROM", not docker commands